.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
```suggestion
  - package-ecosystem: 'npm'
    directory: '/'
    schedule:
      interval: 'daily'
    pull-request-branch-name:
      separator: "-"
    open-pull-requests-limit: 10
```

Reply:
I think there's a couple options we want to include - the limit is just a safety guard so it doesn't go too crazy if there's lots of updates